spec/factories/tenants.rb

```diff
@@ -2,7 +2,7 @@
   factory :tenant, aliases: [:openstack_tenant], class: 'Atmosphere::Tenant' do |f|
     tenant_type 'private'
     tenant_id { SecureRandom.hex(4) }
-    site_id { 'somesite' }
+    site_id { "site_id_#{SecureRandom.hex(4)}" }
```

Comment:
This is a factory, random values are good here. If needed you can assign required value while creating object.